### ISSUE TITLE
Login box reset

### ISSUE THREAD

**Issue by elektrux** [MrRoboman4321/talkio-production#2]
After an a user does something wrong when registering/logging in, the input boxes should go blank.

**Comment by benwiebe** [comment]
I cleared the registration boxes in commit 8ce97e8, but not login. Do we really need to clear login boxes? I think at least keeping the username there until logout would be good

**Comment by elektrux** [comment]
Yeah that's fine. Let's make sure to clear the password, though. I've been having some trouble syncing my github account between computers, so I may not be current.
